site/source/Makefile

```
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = site\source
+BUILDDIR      = site\static\api
```

```suggestion
SOURCEDIR     = site/source
BUILDDIR      = site/static/api
```
This would probably fail on unix.